src/test/java/com/rtb/projectmanagementtool/project/ProjectControllerTest.java

```diff
@@ -1,35 +1,136 @@
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
 import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.rtb.projectmanagementtool.project.ProjectController;
+import com.rtb.projectmanagementtool.project.ProjectData;
+import java.util.ArrayList;
+import java.util.Arrays;
 import org.junit.After;
+import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
+import org.junit.Test;
 
 public class ProjectControllerTest {
   private final LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  DatastoreService ds;
+  ProjectController projectController;
+
+  // Project Creators
+  private final Long USER1 = 1l;
+  private final Long USER2 = 2l;
+  private final Long USER3 = 3l;
+  private final Long USER4 = 4l;
+
+  private ProjectData PROJECT1 = new ProjectData("Project 1", "Project 1 Description", USER1);
+  private ProjectData PROJECT2 = new ProjectData("Project 2", "Project 2 Description", USER1);
+  private ProjectData PROJECT3 = new ProjectData("Project 3", "Project 3 Description", USER2);
+  private ProjectData PROJECT4 = new ProjectData("Project 4", "Project 4 Description", USER3);
 
   @Before
   public void setUp() {
     helper.setUp();
+
+    ds = DatastoreServiceFactory.getDatastoreService();
+    projectController = new ProjectController(ds);
+
+    // Add users to projects
+    PROJECT2.addAdminUser(USER2);
+    PROJECT2.addRegularUser(USER4);
+    PROJECT2.addAdminUser(USER3);
+    PROJECT1.addAdminUser(USER4);
+    PROJECT4.addRegularUser(USER2);
+    PROJECT3.addRegularUser(USER3);
+    PROJECT3.addRegularUser(USER1);
+
+    // Set hardcoded ids to mimic expected behavior
+    PROJECT1.setId(1l);
+    PROJECT2.setId(2l);
+    PROJECT3.setId(3l);
+    PROJECT4.setId(4l);
   }
 
   @After
   public void tearDown() {
     helper.tearDown();
   }
 
-  @Ignore
-  public void noProjects() {}
+  // One test method because I need tests to run in this order
+  @Test
+  public void runProjectControllerTest() {
+    testAddProjectsToDatabase();
+    testSaveProjectsToDatabase();
+    testGetProjectWithUserAndCreator();
+    testRemoveProjects();
+  }
+
+  // Test saving projects to database
+  // Projects added here are used in subsequent test methods
+  public void testAddProjectsToDatabase() {
+    // Add projects to database to use get projects later
+    projectController.addProject(PROJECT1);
+    Assert.assertEquals(1, projectController.getAllProjects().size());
+
+    projectController.addProject(PROJECT2);
+    Assert.assertEquals(2, projectController.getAllProjects().size());
```

Comment:
You don't necessarily have to assert every time you add (or maybe you don't have to invoke 4 times).  Conceptually the 3rd and 4th addProject calls don't differ from the first 2 and I can't think of a benefit for doing them that much. 

Doing twice is a good practice though, the second addProject confirms the previously added items were not overwritten.

Maybe as an alternative:  add 2 -> assert -> add 2 more -> assert again